? test/browser_find_node.cpp
? test/parse/testsuite/lib
? test/parse/testsuite/parse.good/route-referencing-current-node.wrl
Index: configure.ac
===================================================================
RCS file: /cvsroot/openvrml/openvrml/configure.ac,v
retrieving revision 1.68.2.5
diff -u -r1.68.2.5 configure.ac
--- configure.ac	30 Dec 2004 01:34:00 -0000	1.68.2.5
+++ configure.ac	9 Jan 2005 09:13:35 -0000
@@ -165,10 +165,10 @@
   AC_DEFINE([OPENVRML_ENABLE_SCRIPT_NODE_JAVASCRIPT], [1],
             [Defined if Script node JavaScript support is enabled.])
   GRE_PATH=`$PKG_CONFIG --libs-only-L mozilla-js | sed -e 's/^-L//' -e 's/ *$//'`
-  if test -n "${OPENVRML_RPATH}" -a -n "${GRE_PATH}"; then
-    OPENVRML_RPATH=":${OPENVRML_RPATH}"
+  if test -n "${openvrml_run_path}" -a -n "${GRE_PATH}"; then
+    openvrml_run_path=":${openvrml_run_path}"
   fi
-  OPENVRML_RPATH="${GRE_PATH}${OPENVRML_RPATH}"
+  openvrml_run_path="${GRE_PATH}${openvrml_run_path}"
 fi
 AC_SUBST([JS_CFLAGS])
 AC_SUBST([JS_LIBS])
@@ -250,6 +250,9 @@
 # Optional dependencies (substituted in the pkg-config metadata files).
 #
 AC_SUBST([OPENVRML_PKG_REQUIRES])
+if test -n "${openvrml_run_path}"; then 
+  OPENVRML_RPATH="-rpath ${openvrml_run_path}"
+fi
 AC_SUBST([OPENVRML_RPATH])
 
 AC_ARG_VAR([XPIDL], [Mozilla XPCOM IDL compiler])
Index: src/lookat/Makefile.am
===================================================================
RCS file: /cvsroot/openvrml/openvrml/src/lookat/Makefile.am,v
retrieving revision 1.19
diff -u -r1.19 Makefile.am
--- src/lookat/Makefile.am	26 Jan 2004 18:32:20 -0000	1.19
+++ src/lookat/Makefile.am	9 Jan 2005 09:13:35 -0000
@@ -2,7 +2,7 @@
         -I$(top_srcdir)/src/libopenvrml \
         -I$(top_srcdir)/src/libopenvrml-gl
 AM_CXXFLAGS = @SDL_CFLAGS@
-AM_LDFLAGS = -rpath @OPENVRML_RPATH@ @SDL_LIBS@ @JNI_LIBS@
+AM_LDFLAGS = @OPENVRML_RPATH@ @SDL_LIBS@ @JNI_LIBS@
 LDADD = $(top_builddir)/src/libopenvrml-gl/libopenvrml-gl.la \
         $(top_builddir)/src/libopenvrml/libopenvrml.la
 
Index: test/Makefile.am
===================================================================
RCS file: /cvsroot/openvrml/openvrml/test/Makefile.am,v
retrieving revision 1.6.2.1
diff -u -r1.6.2.1 Makefile.am
--- test/Makefile.am	30 Nov 2004 07:21:41 -0000	1.6.2.1
+++ test/Makefile.am	9 Jan 2005 09:13:35 -0000
@@ -1,7 +1,7 @@
 SUBDIRS = parse
 
 AM_CPPFLAGS = -I$(top_srcdir)/src/libopenvrml
-AM_LDFLAGS = -rpath @OPENVRML_RPATH@
+AM_LDFLAGS = @OPENVRML_RPATH@
 LDADD = $(top_builddir)/src/libopenvrml/libopenvrml.la
 
 if ENABLE_TESTS
Index: test/parse/Makefile.am
===================================================================
RCS file: /cvsroot/openvrml/openvrml/test/parse/Makefile.am,v
retrieving revision 1.1.2.1
diff -u -r1.1.2.1 Makefile.am
--- test/parse/Makefile.am	30 Nov 2004 07:21:42 -0000	1.1.2.1
+++ test/parse/Makefile.am	9 Jan 2005 09:13:35 -0000
@@ -1,7 +1,7 @@
 SUBDIRS = testsuite
 
 AM_CPPFLAGS = -I$(top_srcdir)/src/libopenvrml
-AM_LDFLAGS = -rpath @OPENVRML_RPATH@ @JNI_LIBS@
+AM_LDFLAGS = @OPENVRML_RPATH@ @JNI_LIBS@
 LDADD = $(top_builddir)/src/libopenvrml/libopenvrml.la
 
 if ENABLE_TESTS
